src/main/java/jrm/service/UserAuthApiService.java

```diff
@@ -25,6 +25,12 @@
     @Value("${javaroadmap.api.base-v1-url}schedulers/delete-guests-users")
     private String deleteGuestUsers;
 
+    @Value("${javaroadmap.api.base-v1-url}modules")
+    private String mainPage;
```

Comment:
I don't think it's main page. I hope it can be `modulesPage`, or just `modules`.

Reply:
I agree. Fixed.